pkg/security/module/cws.go

```diff
@@ -328,10 +330,12 @@ func (c *CWSConsumer) LoadPolicies(policyProviders []rules.PolicyProvider, sendL
 	// load policies
 	c.policyLoader.SetProviders(policyProviders)
 
+	policySet := c.probe.NewPolicySet(c.getEventTypeEnabled())
 	// standard ruleset
-	ruleSet := c.probe.NewRuleSet(c.getEventTypeEnabled())
+	ruleSet := policySet.RuleSets[rules.DefaultRuleSetName]
```

Comment:
I would create here 3 ruleset using different filter : 

- one global without any tags filter. This one would be used to generate the policy loaded.
- the second one for the probe evaluation excluding the threat score tag
- The third one for the threat evaluation excluding the non threat score tag